realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -510,8 +510,9 @@ public long upperBoundLong(long columnIndex, long value) {
     public TableView findAllLong(long columnIndex, long value){
         // Execute the disposal of abandoned realm objects each time a new realm object is created
         context.executeDelayedDisposal();
-        long nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
-        try { 
+        long nativeViewPtr = 0;
+        try {
+            nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
```

Comment:
@beeender You're right. I just looked into `CATCH_STD()` and realized that this change isn't actually covering anything. The `native*` calls can live outside as is now. 

Thanks for the pointer!